### PR TITLE
LPS-141760 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-141760

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by ozysouza** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 1]
```diff
@@ -447,6 +447,49 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-137902. The user can remove a localized content in the web content."
+	@priority = "4"
+	test RemoveLocalizedContent {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:452]: test DeleteLocalizedContent

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 2]
```diff
@@ -447,6 +447,49 @@ definition {
...
+				webContentTitle = "Título del contenido web");
+		}
+
+		task ("Remove the Spanish translation in the content field") {
+			WebContent.addCP(webContentContent = "");
+
+			PortletEntry.publish();
+		}
+
+		task ("View Spanish content field is empty after published") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:482]: "View content field is empty in Spanish after publish"

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind mdify according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind mdify according to comments? Thanks

Hi @Tim-Cao, Sure! PR Updated. Would you have a look at? Thanks!

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**